src/test/scala/com/microsoft/hyperspace/goldstandard/TPCDSBase.scala

```diff
@@ -0,0 +1,575 @@
+// scalastyle:off
+
```

Comment:
To Reviewers: This file is build after modifying spark's https://github.com/apache/spark/blob/branch-3.1/sql/core/src/test/scala/org/apache/spark/sql/TPCDSBase.scala

Reply:
Note: I have kept only TPCDS Query support for query version 1.4.

Removed support for "modified_queries" and "version 2.7" from the original spark file